File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/entity/SellerSubmitStatus.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.entity;

public enum SellerSubmitStatus {
    PENDING,
    APPROVED,
    REJECTED;

    public boolean isPending() {
        return this == PENDING;
    }

    public boolean isApproved() {
        return this == APPROVED;
    }
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/entity/SellerSubmitStatus.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.entity

enum class SellerSubmitStatus {
    PENDING,
    APPROVED,
    REJECTED;

    val isPending: Boolean
        get() = this == PENDING

    val isApproved: Boolean
        get() = this == APPROVED
}
